renode-linux-runner-docker/payload/run-in-renode.py

```diff
@@ -19,7 +19,35 @@
 from subprocess import run, DEVNULL, CalledProcessError
 from sys import stdout as sys_stdout, exit as sys_exit, argv as sys_argv
 from time import sleep
+from re import sub as re_sub, compile as re_compile
 
+CR = r'\r'
+
+class FilteredStdout(object):
+    """
+    Stdout wrapper which replaces found pattern with 'replace' string.
```

Comment:
This comment is good, but the class itself has nothing to do with `\r`. That is specified at the object creation site (which is a good approach). Let's move the `\r` explanation to there.